install.sh

```diff
@@ -340,6 +340,7 @@ function print_the_help {
   echo "USAGE:  ./eic-shell [OPTIONS] [ -- COMMAND ]"
   echo "OPTIONAL ARGUMENTS:"
   echo "          -u,--upgrade    Upgrade the container to the latest version"
+  echo "          --noX           Disable X11 forwarding on macOS (not needed for singularity)"
```

Comment:
Remove unnecessary parenthetical (you're in `install_docker`).
```suggestion
  echo "          --noX           Disable X11 forwarding on macOS"
```
It would be nice if you could only print this on macOS, or make this work on linux too by not just greping for Xquartz.

Reply:
I removed the parenthetical, it's not needed since we're in the docker function anyway.
To your second point, in linux/singularity no such trickery is needed, from my testing X11 works just fine out of the box.